frontend/src/components/PictureOfTheDay.tsx

```diff
@@ -5,7 +5,7 @@ import useNasaPicture from "../hooks/useNasaPicture";
 export default function PictureOfTheDay (){
     const pictureOfTheDay = useNasaPicture()
     if(!pictureOfTheDay){
-        return <div>Picture is not available</div>
+        return <div></div>
```

Comment:
Vielleicht könntest du hier einen Hinweis einbauen, dass kein Bild verfügbar ist. Oder evtl ein "navigate to" 
-ah, dass hattest du vorher sogar schon.